assembl/lib/frontend_urls.py

```diff
@@ -23,6 +25,27 @@
 }
 
 
+frontend_routes = None
+
+
+def get_frontend_urls():
+    """Get all V2 routes from source of truth"""
+
+    current = dirname(__file__)
+    route_path = join(current, '..', 'static2/routes.json')
+    if not exists(route_path):
+        raise IOError("Route path could not be found")
+    with open(route_path) as paths:
+        routes = json.loads(paths.read())
```

Comment:
If this function is called often then there can be an efficiency problem. For example, reading and parsing a file everytime we want to build an URL or know how to interpret an URl can result in too much I/O on disk. If it is the case, probably read the file once in the app to save routes in memory and then refer to them?